Tests/Adapter/PHPCR/PHPCRAdapterTest.php

```diff
@@ -5,17 +5,16 @@
 use PHPUnit\Framework\TestCase;
 use Vich\UploaderBundle\Adapter\PHPCR\PHPCRAdapter;
 use Vich\UploaderBundle\Tests\DummyEntity;
+use Doctrine\Common\Persistence\Event\LifecycleEventArgs;
```

Comment:
This is violating "ordered_imports" rule of .php-cs-fixer (unfortunately, I can't control settings on StyleCI so it's not automatically detected)